docs/audit_report/changes/topics/keccak.yml

```diff
@@ -37,3 +37,8 @@ patches:
 - commit: cf045523e2fbdeccf99e6f5e546dbd41cddf851f  # https://github.com/randombit/botan/commit/cf045523e2fbdeccf99e6f5e546dbd41cddf851f
   classification: info
   auditer: reneme
+
+# Refactor: Remove MDx_HashFunction  (@reneme)
+- pr: 3705  # https://github.com/randombit/botan/pull/3705
+  merge_commit: 1db2b19cd07d88630159716fbb6ebe7fd5a5dfdf
+  classification: relevant
```

Comment:
That doesn't touch Keccak. The need for this came up while developing the Keccak refactoring but it is orthogonal to it. I'd suggest `chores.yml`, because it is an internal refactoring without any influence on the security of the library (given that I didn't break anything).